lang/image.bzl

```diff
@@ -224,6 +226,7 @@ def _app_layer_impl(ctx, runfiles = None, emptyfiles = None):
         directory = "/",
         file_map = file_map,
         empty_files = empty_files,
+        empty_dirs = emptydirs,
```

Comment:
Shouldn't we perform checks on these emptydirs (as is done for emptyfiles in lines 195-199) to verify there are no collisions between passed args and other files/dirs?